### PR TITLE
Sentry: add geo, datasets, and conformal

### PR DESCRIPTION
**Pull request by jerneju**
Sentry: add geo, datasets, and conformal

### CODE REVIEW

File: error_report/sentry.py [hunk 1]
```diff
@@ -33,16 +33,19 @@
...
+    'educational':      ('Orange3-Educational', "https://93323bc17a094974a830b25abbae01b5:4fd5e7c529e34afd97ceca08ed4f059d@sentry.io/167545"),
+    'geo':              ('Orange3-Geo', "https://f3b7d23593d14247808b70ff964b3956:ff25c1d23d3a4eca849429c731c874d9@sentry.io/167528"),
+    'imageanalytics':   ('Orange3-ImageAnalytics', "https://cc2ef6171aad4b6ba344e2851169db7d:cd21ed3e80ae4f4385b31a24e0d036cf@sentry.io/161064"),
+    'network':          ('Orange3-Network', "https://14706c0ff3e047d999cff64e6100eb25:1dd7b84d0afc449abba1757e3520b0c2@sentry.io/167534"),
+    'prototypes':       ('Orange3-Prototypes', "https://d7440097e7f64e4cbff90dd31fc8876e:dde09f7ba917431884b7eb04c814b824@sentry.io/167530"),
+    'recommendation':   ('Orange3-Recommendation', "https://e447ddb4e80149289bca679121359c03:e4b9a0f1a1414f7d906e56b8e28be9cc@sentry.io/167543"),
+    'text':             ('Orange3-Text', "https://38ffabded40c46b9952b2acebc726866:147d6a5becfa40499b6d79e858fb6ef1@sentry.io/128443"),
+    'timeseries':       ('Orange3-Timeseries', "https://e8f30f9dbaf74635bb10e37abe0b5354:2478a41e2f95463db8ceebfeb060cc99@sentry.io/161065")
```
Comment (kernc) [inline error_report/sentry.py]: I beg to wonder, are these basic authentication parts possibly secrets?

Comment (jerneju) [inline error_report/sentry.py]: Yes, public secrets. If they are abused, they will be changed and hidden.

Comment (astaric) [inline error_report/sentry.py]: As Sentry lists both DSN and Public DSN, I would use public one here.

https://sentry.io/biolab/orange-bio/getting-started/

Comment (astaric) [inline error_report/sentry.py]: (if possible, otherwise these are ok)

Comment (jerneju) [inline error_report/sentry.py]: Tried Public DSN but does not work. At least not for the main project.

**Pull request merged**